frontend/src/metabase/visualizations/visualizations/LinkViz/LinkViz.unit.spec.tsx

```diff
@@ -201,6 +202,43 @@ describe("LinkViz", () => {
   });
 
   describe("entity links", () => {
+    it("wraps the dashboard's description", async () => {
```

Comment:
1. :x: This test does not have assertions for what was broken - please turn it into actual repro
2. We add `(metabase#1234)` tags to test names that are repros ([like this](https://github.com/metabase/metabase/blob/92d615073a3ba31710e26be9f2e83141352d1d1c/frontend/src/metabase/visualizations/components/ChartCaption.unit.spec.tsx#L71))

Reply:
1. should this be an e2e test then? Alternatively we cant check that a max-width has indeed been set on the component, but that would not really test the issue (whatever the max-width is, there can always be a smaller viewport I guess)
 2. yep, didn't know that, will change

Reply:
> should this be an e2e test then? 

Yes, we need to assert that tooltip does not overflow the viewport - we can't use unit tests for this as elements don't have dimensions there.

>  Alternatively we cant check that a max-width has indeed been set on the component, but that would not really test the issue (whatever the max-width is, there can always be a smaller viewport I guess)

Yeah, `max-width` assertion should be treated as "extra" - the main problem is the overflow.